src/operator/subgraph/subgraph_property.h.bak

```diff
@@ -0,0 +1,399 @@
+/*
```

Comment:
commit by mistake?

Reply:
You're right, will remove backup files :)